js/templates/pageNav.html

```diff
@@ -30,15 +30,15 @@
       <div>
         <div class="flexVCent box">
           <a class="iconBtn ion-android-notifications js-navNotifBtn"></a>
-          <a id="AvatarBtn" class="discSm border clrBr js-navListBtn navListBtn"
+          <a id="AvatarBtn" class="discSm border clrBr2 clrSh1 js-navListBtn navListBtn"
           <% if (ob.avatarHash) { %>
           //this will be updated to the real image parameter
           style="background-image: url(<%= ob.getServerUrl(`ipfs/${ob.avatarHash}`) %>), url('../imgs/defaultAvatar.png')"
           <% } else { %>
             style="background-image: url('../imgs/defaultAvatar.png')"
           <% } %>
           ></a>
-          <nav class="navList listBox clrS clrSh1 foldDown js-navList js-navPopMenu">
+          <nav class="navList listBox clrP clrSh1 clrBr foldDown js-navList js-navPopMenu">
```

Comment:
It could just be a timing thing. I can play around with it a bit to see if I can improve on it.